src/components/app/MenuButtons/index.css

```diff
@@ -27,16 +26,62 @@
   background-color: rgba(0, 0, 0, 0.2);
 }
 
+/* Use this class if a button has an icon */
+.menuButtonsButtonButton__hasIcon::before {
+  /* The icon size and width and height are defined as 1em so that it's adjusted automatically
+   * with the font-size. Then it has 4px padding on the right side (because it's
+   * at the left of the content), and the image is at the left of this space.
+   * So here is the layout for this element:
+   * <-left padding-> <-ICON (1em)-> <-4px-> CONTENT
+   */
+  width: 1em;
```

Comment:
FTR I used `1em` initially, not so that it would resize when zooming, but so that if we change the font-size of an element this would adjust automatically without needing that we change the value everywhere else.

I defined them all to 12px, also also reworked the existing one for the Docs button to the 12px grid. Now the icon sizes are 12px despite the font-size being 11px, but this still looks good in my opinion so I didn't change the font-size.

TBH I don't really see the difference from the previous version :-)